post-processor/amazon-import/post-processor.go

```diff
@@ -87,6 +92,11 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 		}
 	}
 
+	if !(p.config.Format == "ova" || p.config.Format == "raw" || p.config.Format == "vmdk" || p.config.Format == "vhd" || p.config.Format == "vhdx") {
```

Comment:
For readability; I'd make of this check a switch statement; something like:

```suggestion
	switch p.config.Format {
	case "ova", "raw", "vmdk", "vhd", "vhdx":
	default:
```